docs/chapters/04_features.md

```diff
@@ -317,6 +317,69 @@ And this would be the response:
 
 > [!NOTE] Remember to set the proper **access token** for secured commands, check ["Authorizing operations"](#authorizing-operations).
 
+### Adding "before" hooks to your commands
+As seen above, when you send commands, you can include input parameters to be received for the command handler. Although, there are some cases where you'd want to do some actions just before the `handle` function gets called in your command, like:
+- Validating parameters
```

Comment:
Not sure if we can reuse the docs in the previous PR https://github.com/boostercloud/booster/pull/850 so it's everything in the same chapter

Reply:
I'm not sure. I'd keep them separated in each section, since the hook arguments are different between ReadModels and Commands

Reply:
Agree with @davidverdu